airflow/operators/python.py

```diff
@@ -380,6 +390,8 @@ def __init__(
                 self.requirements.append('lazy-object-proxy')
             if self.use_dill and 'dill' not in self.requirements:
                 self.requirements.append('dill')
+        # Only allows to use clone_virtualenv_packages if no python version is specified
+        self.clone_virtualenv_packages = clone_airflow_virtualenv if not python_version else False
```

Comment:
I think we should

1. Raise an exception if both arguments are set
2. Maybe ignore `python_version` if it matches `sys.version_info`...?

Reply:
AH yeah - that's the "previous" PR (this one is based on it) - I will come back to it shortly and address the comments there :)

Reply:
It's the #19189 - but I will take your comments and answer here and address them there.